src/ppom-controller.ts

```diff
@@ -304,27 +318,18 @@ export class PPOMController extends BaseControllerV2<
         : fileFetchScheduleDuration;
     this.#securityAlertsEnabled = securityAlertsEnabled;
     this.#blockaidPublicKey = blockaidPublicKey;
-    this.#ppomInitialisationCallback = ppomInitialisationCallback;
-
-    // add new network to chainStatus list
-    onNetworkChange(this.#onNetworkChange.bind(this));
 
     // enable / disable PPOM validations as user changes preferences
     onPreferencesChange(this.#onPreferenceChange.bind(this));
 
     // register message handlers
     this.#registerMessageHandlers();
 
+    // subscribe to events
+    this.#subscribeMessageEvents();
```

Comment:
Q: are there any parts of this lifecycle we should consider unsubscribing to the events?

Reply:
Good point, un-subscribing should not be needed for network controller.